saleor/checkout/models.py

```diff
@@ -18,7 +18,10 @@
 from ..giftcard.models import GiftCard
 from ..shipping.models import ShippingMethod
 
-CENTS = Decimal("0.01")
+if TYPE_CHECKING:
+    from ..product.models import ProductVariant
+    from django_measurement import Weight
```

Comment:
I am not sure if `isort` handled that. Maybe we could separate 3rd packages from our modules? Or just put it above the `if`

Reply:
Related issue: https://github.com/timothycrosley/isort/issues/818

Reply:
Maybe laziness is speaking thru me, but unless any tool does not manage that, I don't want to go back to pre isort/black time and deal with it manually :/ Type checking sections tend to be way smaller than usual imports so maybe it's not an issue?